infra/executor/src/main/java/org/apache/shardingsphere/infra/executor/sql/log/SQLLogger.java

```diff
@@ -44,7 +44,7 @@ public final class SQLLogger {
      */
     public static void logSQL(final QueryContext queryContext, final boolean showSimple, final ExecutionContext executionContext) {
         log("Logic SQL: {}", queryContext.getSql());
-        log("SQLStatement: {}", queryContext.getSqlStatementContext().getSqlStatement());
+        log.debug("SQLStatement: {}", queryContext.getSqlStatementContext().getSqlStatement());
```

Comment:
How about move it to `logNormalMode`?

Reply:
Sorry, ShardingSphere is a public project, it should not upon to one person's requirement.

Reply:
I created a new issue to discuss about remove it. #22453

Reply:
> I created a new issue to discuss removing it. #22453

Okay, thanks.